### DISCUSSION TITLE
Webpack5 splitChunks  'initial' -> bundle.js not working

### DISCUSSION THREAD

**Discussion by kapilgp** [webpack/webpack#16967] [Q&A] [answered]
# Bug report

**What is the current behavior?**
As soon as i am defining splitChunks.chunks: 'initial'
the produced bundle is not being executed in the browser -> not working.
If i define it as 'async' it is working.
Also with webpack 4 it was working but with webpack 5 it isn't working.

**If the current behavior is a bug, please provide the steps to reproduce.**
Configure splitChunks.chunks: 'initial'

If it helps you i can send you the produced bundle.

**What is the expected behavior?**
The browser bundle is working normally. As it also working fine with Webpack4

**Other relevant information:**
webpack version:  ^5.77.0
Node.js version: v16.15.0
Operating System: ubuntu
Additional tools:

**Comment by alexander-akait** [comment]
> If the current behavior is a bug, please provide the steps to reproduce.

Sorry it is not enought to help you, provide more information - full configuration and an example of code with the problem

**Reply by kapilgp** [reply]
Hopefully it helps you

optimization: {
        usedExports: true,
        splitChunks: { // CommonsChunkPlugin()
            name: false,
            **chunks:"initial",**
            maxAsyncRequests: 1,
            maxInitialRequests: 1,
            cacheGroups: {
                // utils: {
                //     test: /[\\/]utils[\\/]/,
                //     name: 'utils',
                //     chunks: 'all'
                // },
                common: {
                    test: /[\\/]node_modules[\\/](create-react-class|prop-types|style-loader|react|history|es6-promise|query|object-assign|process|fbjs|invariant|warning|strict|hoist)|plugins[\\/]zepto.min|girnarsoft-react-common[\\/]utils[\\/](ReactLazy|api|detect|global.js$)|data[\\/]menus/,
                    name: 'common',
                    chunks: 'all',
                    priority: -10,
                    reuseExistingChunk: true,
                }
            }
        },

**Reply by alexander-akait** [reply]
Sorry, not, I don't know what is not working, that is why I asked to provide an example of code (any, with the problem), you can create small reproducible example using github

**Comment by sirrodgepodge** [comment]
I'm having this same issue... the output file from webpack is literally not executing (even a console log at the top of the file) when the output file is loaded in the browser if I change "chunks" to "initial" or "all"

**Comment by kapilgp** [comment, marked as answer]
It resolved for me after upgrade React Router 5
